src/tests/class_tests/openms/source/MRMFeatureSelector_test.cpp

```diff
@@ -47,23 +54,30 @@ START_TEST(MRMFeatureSelector, "$Id$")
 /////////////////////////////////////////////////////////////
 /////////////////////////////////////////////////////////////
 
-MRMFeatureSelector* ptr = 0;
-MRMFeatureSelector* null_ptr = 0;
+MRMFeatureSelectorScore* ptr = 0;
+MRMFeatureSelectorScore* null_ptr = 0;
+
+const String features_path = OPENMS_GET_TEST_DATA_PATH("MRMFeatureSelector_150601_0_BloodProject01_PLT_QC_Broth-1_1.featureXML");
+
+FeatureMap feature_map;
```

Comment:
These are not ok.

It's hard to keep track of state for these variables (any non `const` container, in general).
Better move these declaration inside the test sections. We duplicate code, true, but it gives us safer code. It makes us sure that we're dealing with clean variables and data.